gateway-ha/src/main/java/io/trino/gateway/ha/router/HaGatewayManager.java

```diff
@@ -24,6 +24,7 @@
 import java.util.List;
 import java.util.Optional;
 
+import static com.google.common.base.Preconditions.checkState;
```

Comment:
The commit title is too long. Please follow https://trino.io/development/process#pull-request-and-commit-guidelines-